packages/office-addin-sso/src/commands.ts

```diff
@@ -17,6 +18,19 @@ export async function configureSSO(manifestPath: string) {
   if (process.platform !== "win32" && process.platform !== "darwin") {
     console.log(chalk.yellow(`${process.platform} is not supported. Only Windows and Mac are supported`));
     return;
+  } else if (applicationDataConfigured(manifestPath)) {
+    console.log(
+      chalk.yellow("Project has already been at least partially configured.  Can't setup new configuration completely.")
```

Comment:
Please check with the content team about the phrasing here. It's missing articles like "the" and I think the tone could be improved.

Reply:
Anyone specific on the content team I should be asking?

Reply:
Not anyone specific, but @millerds I will send to you in private some contacts of the contents team

Reply:
Updated the text to be similar to the existing text.